apps/prairielearn/src/lib/assessment.sql

```diff
@@ -704,7 +704,10 @@ WITH
         LEFT JOIN rubric_gradings AS rg ON (rg.id = gj.manual_rubric_grading_id)
       WHERE
         iq.assessment_instance_id = $assessment_instance_id
-        AND gj.grading_method = 'Manual'
+        AND (
+          gj.grading_method = 'Manual'
+          OR gj.grading_method = 'AI'
+        )
```

Comment:
This is cleaner and, if an index is involved (not the case here, but who knows), it's more likely to satisfy the index.
```suggestion
        AND gj.grading_method IN ('Manual', 'AI')
```